framework/src/main/java/fi/otavanopisto/pyramus/koski/KoskiStudentHandler.java

```diff
@@ -172,7 +172,8 @@ protected void removeInternetixOid(KoskiStudyProgrammeHandler handler, Student s
     
     ObjectMapper mapper = new ObjectMapper();
     try {
-      userVariableDAO.setUserVariable(student, KOSKI_INTERNETIX_STUDYPERMISSION_ID, mapper.writeValueAsString(oids));
+      String variableValue = CollectionUtils.isNotEmpty(oids) ? mapper.writeValueAsString(oids) : "";
```

Comment:
Does saveOrValidateInternetixOid in this class need the same check?

Reply:
Added